lib/versioning/node/schedule.ts

```diff
@@ -11,5 +11,5 @@ interface NodeJsSchedule {
 export type NodeJsData = Record<string, NodeJsSchedule>;
 
 export const nodeSchedule: NodeJsData = JSON.parse(
-  dataFiles.get('data/node-js-schedule.json')
+  dataFiles.get('data/node-js-schedule.json') ?? ''
```

Comment:
```suggestion
  dataFiles.get('data/node-js-schedule.json')!
```
This should never undefined or null.

Not sure about `!`. Maybe needs a string cast.

Reply:
Should I ignore it for the time-being?

Reply:
non-null assertion is disabled in eslint config.

Reply:
We can disable it here for now

Reply:
Have added a different and returning an empty string.

`nodeSchedule` is used here, incase empty object is returned the function returns false, which seems the expected outcome incase we can't get version information.
https://github.com/renovatebot/renovate/blob/77708881845c9520e29e878db66830d88a5467c5/lib/versioning/node/index.ts#L32-L42

Reply:
> `nodeSchedule` is used here, incase empty object is returned the function returns false, which seems the expected outcome incase we can't get version information.
> 
> https://github.com/renovatebot/renovate/blob/77708881845c9520e29e878db66830d88a5467c5/lib/versioning/node/index.ts#L32-L42
> 
> So implemented a solution that returns empty object incase the string imported from the file is null.

this causes coverage issues reverting to non-null assertion

Reply:
above solution causes coverage issues which can only be solved by adding new test which uses a mock of `../../data-files.generated` . Reverting to using non-null assertion.